<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowerGui</class>
 <widget class="QMainWindow" name="BorrowerGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/inori_zirong_chibi.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblHeader">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>146</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ff0000;&quot;&gt;Borrower&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>644</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOverdue">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>Show Overdue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>Add Borrower</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>981</width>
      <height>571</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>979</width>
       <height>569</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>421</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/greenBackGround.jpg);</string>
      </property>
      <widget class="QLabel" name="lblBRNAME">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblBRTYPE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblBRDEP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Department&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblPHONE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Phone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnBrName">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>221</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnBrPhone">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>221</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbBrType">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbBrDepartment">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>240</y>
         <width>211</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTableView" name="tbBorrower">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>961</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/White-Wallpapers.jpg);</string>
      </property>
     </widget>
     <zorder>tbBorrower</zorder>
     <zorder>widget</zorder>
    </widget>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>lblHeader</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
